_rules/block-collapsible-3e12e1.md

```diff
@@ -46,7 +46,9 @@ _There are no major accessibility support issues known for this rule._
 
 Usually the same [instrument][] removes both [visibility][visible] and [inclusion in the accessibility tree][included in the accessibility tree] of a [block of repeated content][]. That [instrument][] may remove all [blocks of repeated content][block of repeated content]. If there is no [block of repeated content][] before the non-repeated content the rule passes.
 
-[Technique SCR28: Using an expandable and collapsible menu to bypass block of content][tech scr28] does not have any requirements concerning the location of the [instruments][instrument] in relation to the [block of repeated content][] they control, hence this rule doesn't. It is likely a good idea to either keep each [instrument][] close to the start of the [block of repeated content][] it controls; or to group them all in one place near the start of the document. Notably, [instruments][instrument] located after (in reading order) the block they collapse are likely not satisfying [Success Criterion 2.4.1 Bypass blocks][sc241], which this rule is designed for. Thus, it is possible to pass this rule without satisfying [Success Criterion 2.4.1 Bypass blocks][sc241].
+[Technique SCR28: Using an expandable and collapsible menu to bypass block of content][tech scr28] does not have any requirements concerning the location of the [instruments][instrument] in relation to the [block of repeated content][] they control, hence this rule doesn't. It is likely a good idea to either keep each [instrument][] close to the start of the [block of repeated content][] it controls; or to group them all in one place near the start of the document.
+
+[Instruments][instrument] located after the block they collapse in reading order are likely not satisfying [Success Criterion 2.4.1 Bypass blocks][sc241], which this rule is designed for. Thus, it is possible to pass this rule without satisfying [Success Criterion 2.4.1 Bypass blocks][sc241].
```

Comment:
> This seems like it could possibly go in the assumptions section. E.g.,
> 
> `We assume the instrument is located before the block it collapses. If the instrument is located later in the reading order, it is likely not satisfying [Success Criterion 2.4.1 Bypass blocks][sc241], Thus, it is possible to pass this rule without satisfying [Success Criterion 2.4.1 Bypass blocks][sc241].`
> 
> But maybe the "likely not" makes the statement too weak. I think it could be up for discussion but would be fine leaving it as is.

@Jym77  Would you be comfortable with me trying to make this an assumption?

Reply:
> I do not think this really is an assumption **made by the rule**. The rule does not assume anything about the position of the instrument. It's quite the contrary, the rule accepts instruments located anywhere on the page, which is likely too lax.
> 
> This is more of a known case where the rule can pass content which is bad (false negative). Maybe it can be rephrased to make this more explicit:
> 
> > The rule accepts instruments located anywhere in the page. Instruments located after the block they collapse …
> 
> Maybe we can also drop the "in reading order" precision which makes the sentence hard to parse. Given that this is in a non-normative note, it is OK to just say "after the block" even if it is a bit ambiguous.

Changed to "sequentially after the block". I think that is what we want to underline here, as this is likely to affect tab order mostly.